components/project-delete-form.js

```
@@ -0,0 +1,75 @@
+        setDeleteOpen(true);
+    }
+
+    useEffect(() => {
+        if (state?.message || state?.success) {
+            setDeleteOpen(false);
+            setOpen(true);
+
+            setTimeout(() => {
+                router.replace('/projects');
+            }, 2000);
+        } else if (state?.message) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)